README.md

```diff
@@ -27,18 +27,27 @@ pip3 install .
 
 You will need to instantiate an AWS Connection:
 ```python
-from hip_data_tools.authenticate import AwsConnection
+from hip_data_tools.connect.aws import AwsConnectionManager, AwsConnectionSettings
 
-conn = AwsConnection(mode="assume_role", settings={"profile_name": "default"})
+# to connect using an aws cli profile
+conn = AwsConnectionManager(AwsConnectionSettings(region_name="ap-southeast-2", profile="default"))
 
-# OR if you want to connect using Env Vars:
-conn = AwsConnection(mode="standard_env_var", settings={})
+# OR if you want to connect using the standard aws environment variables
+# (aws_access_key_id, aws_secret_access_key):
```

Comment:
This comment line seems like irrelevant as the code from line 38 to 41 has been removed.